src/client/store/ui/review/hooks/index.ts

```diff
@@ -6,6 +6,24 @@ import { useAppSelector } from '@client/store'
 export const useReviewStatus = (key: string): ReviewStatus =>
   useAppSelector((state) => state.ui.review.status[key] || ({} as ReviewStatus))
 
+export const useOdpReviewSummary = (odpId: number): ReviewStatus => {
+  const statuses = useAppSelector((state) => Object.values(Object.fromEntries(Object.entries(state.ui.review.status))))
+
+  return statuses.reduce(
+    (acc, curr) =>
+      curr.key.startsWith(`odp-${odpId}-`)
```

Comment:
can't the `odp-${odpId}-` be a created by a common method called from Topics? something like Topics.getOdpReviewTopicKeyPrefix ?